glyphs-reader/plist_derive/src/attrs.rs

```diff
@@ -26,3 +29,66 @@ impl FieldAttrs {
         Ok(this)
     }
 }
+
+static IGNORE: &str = "ignore";
+static KEY: &str = "key";
+static FROM_PLIST: &str = "fromplist";
+static ALT_NAME: &str = "alt_name";
+static DOC: &str = "doc";
+
+#[derive(Clone, Default, Debug)]
+pub(crate) struct TryFromTokenizerFieldAttrs {
```

Comment:
so presumably we're keeping both of these traits?

Reply:
...lets not